mock/.env.example

```
@@ -1,15 +1,13 @@
 # Discord Bot Token from the Dev Dashboard
-BOT_TOKEN=your_bot_token_here
+DISCORD_BOT_TOKEN=your_bot_token_here
 
 # MongoDB URI for connecting to the database
 MONGO_URI=local_mongodb_or_remote_mongodb_uri_here
 DB_NAME=database_name_to_use_in_cluster
 
 # Webhook URL for reporting unknown exceptions
 UNKNOWN_EXCEPTION_WEBHOOK_URL=your_webhook_url_here
+HANDLED_EXCEPTION_WEBHOOK_URL=your_webhook_url_here
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HANDLED_EXCEPTION_WEBHOOK_URL key should go before the UNKNOWN_EXCEPTION_WEBHOOK_URL key

(UnorderedKey)